semgrep/semgrep/target_manager_extensions.py

```diff
@@ -42,6 +43,7 @@
     + C_EXTENSIONS
     + GO_EXTENSIONS
     + RUBY_EXTENSIONS
```

Comment:
Why isn't PHP here? Are only certain languages added?

Reply:
I don't know either. This file needs some clean-up:
* It looks like `ALL_EXTENSIONS` should be derived from the dict `_LANGS_TO_EXTS` anyway.
* The languages should be ordered alphabetically.

I'd say don't worry about it.